apps/condo/domains/banking/access/BankIntegration.js

```diff
@@ -4,12 +4,20 @@
 
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 
-async function canReadBankIntegrations ({ authentication: { item: user } }) {
+const { checkBankIntegrationsAccessRights } = require('@condo/domains/banking/utils/accessSchema')
+
+const { BANK_INTEGRATION_IDS } = require('../constants')
+
+async function canReadBankIntegrations ({ authentication: { item: user }, context }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
 
     if (user.isAdmin) return {}
 
+    if (await checkBankIntegrationsAccessRights(context, user.id, [BANK_INTEGRATION_IDS.SBBOL])) return {
+        accessRights_some: { user: { id: user.id }, deletedAt: null }, deletedAt: null,
+    }
+
```

Comment:
Redundant.
I don't see reason to hide BankIntegration records, because they are like enums with few hardcoded records.

```suggestion
    
```